### PR TITLE
Feature/edu benefits application review

### PR DESCRIPTION
**Pull request by FernandoBoza**
## Description
Overview and review of 

- Application Information, 
- Contact Information, 
- Service History, 
- Benefits Selection, 
- and Additional Consideration

## Testing done
[] In Progress
[] Done 

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by micahchiang — DISMISSED** [review]
I think this branch needs to be rebased or master needs to be merged in? There are quite a few changes in here that were in merged into master earlier and are unrelated to this PR.

**Review by FernandoBoza — COMMENTED** [review]
merged after rebasing from master

File: src/applications/my-education-benefits/config/form.js [hunk 1]
```diff
@@ -154,7 +154,17 @@ function phoneSchema() {
 const formConfig = {
   rootUrl: manifest.rootUrl,
   urlPrefix: '/',
-  // submitUrl: '/v0/api',
+  preSubmitInfo: {
+    ...preSubmitInfo,
+    notice: (
```
Comment (micahchiang) [inline src/applications/my-education-benefits/config/form.js:159]: Is this notice needed? It looks like there's an existing, identical notice in the [platform presubmit component](https://github.com/department-of-veterans-affairs/vets-website/blob/b23f6e1a74a5e4bea58552e2dda0a49d847ad262/src/platform/forms/preSubmitInfo.js#L5).

Comment (FernandoBoza) [reply src/applications/my-education-benefits/config/form.js:159]: Hey thanks for catching that but I noticed that the [presubmit component](https://github.com/department-of-veterans-affairs/vets-website/blob/b23f6e1a74a5e4bea58552e2dda0a49d847ad262/src/platform/forms/preSubmitInfo.js#L5) styling doesn't fix the VA standard, or in my app.

By default it's pretty compact, a margin top of 1em is what my mock up shows.  

Screen Shot 2021-09-01 at 10 13 42 AM

Comment (FernandoBoza) [reply src/applications/my-education-benefits/config/form.js:159]: @micahchiang on the left is my component with spacing between NOTE and checkbox, and

right image is the component natural styling.

**Review by micahchiang — APPROVED** [review]

**Review by timwright12 — APPROVED** [review]

File: src/applications/my-education-benefits/config/form.js (hunk 1 above)
Comment (erikphansen) [reply src/applications/my-education-benefits/config/form.js:163]: I agree yours looks nicer. 👍  This seems like a change that should be made to the common presubmit component, though.

**Review by cesaresantiago — APPROVED** [review]

File: src/applications/my-education-benefits/config/form.js (hunk 1 above)
Comment (nathanielbuck) [reply src/applications/my-education-benefits/config/form.js:163]: Hey guys, I believe this styling issue was a side effect of styling overrides we made in the My Education Benefits App. I implemented the confirmation checkbox in this PR without the need to customize the preSubmitInfo notice; I just updated our CSS override.

https://github.com/department-of-veterans-affairs/vets-website/pull/18437

**Pull request merged**